cms/migrations/0019_set_pagenode.py

```diff
@@ -63,6 +64,8 @@ class Migration(IrreversibleMigration):
 
     def apply(self, project_state, schema_editor, collect_sql=False):
         connection = schema_editor.connection
+        if not router.allow_migrate(connection.alias, self.app_label):
```

Comment:
Please add a comment as to why this has been added, that way if this causes anyone issue in the future your use case will still be accommodated in any workarounds..